mmdet3d/models/__init__.py

```diff
@@ -30,3 +33,11 @@
     'build_fusion_layer', 'build_model', 'build_middle_encoder',
     'build_voxel_encoder'
 ]
+
+pwd = os.getcwd()
```

Comment:
please use `custom_imports = dict(imports=['projects.detr3d.mmdet3d_plugin'])` in config file to avoid such import approach, following  https://github.com/open-mmlab/mmdetection3d/pull/2170/files#diff-693bc44c728cd63f1c0713a417217f1e56044f4bec431e4054b830a243788da8R6